src/mds/CDir.cc

```diff
@@ -2421,6 +2421,13 @@ void CDir::_omap_commit_ops(int r, int op_prio, int64_t metapool, version_t vers
   auto commit_one = [&](bool header=false) {
     ObjectOperation op;
 
+    /*
+     * just skip submiting the invalid request, which could cause
+     * the cephfs to become readonly.
+     */
+    if (!header && _set.empty() && _rm.empty())
```

Comment:
Oh, RADOS treats an empty "op" as invalid?

Reply:
Yeah. From the logs it is.